### PR TITLE
Implementing ASE without the motion-capture component

### PR DESCRIPTION
**Pull request by dtch1997**

### CODE REVIEW

**Comment by dtch1997** [comment]
Some TODOs:

- In the original paper, encoder (and discriminator) reward is provided to the policy as its reward. I have yet to implement this in the CleanRL version. 
- In the official implementation, `torch.sum` was used to compute encoder loss instead of `torch.mean`, along with `enc_coef = 5`.
